org.lflang/src/org/lflang/generator/c/CConstructorGenerator.java

```diff
@@ -1,8 +1,7 @@
 package org.lflang.generator.c;
 
-import org.lflang.federated.generator.FederateInstance;
 import org.lflang.generator.CodeBuilder;
-import org.lflang.lf.ReactorDecl;
+import org.lflang.lf.Reactor;
```

Comment:
Discussed in person, but I'll answer just in case the question comes up again. This prevents generating code multiple times for the same reactor class just because it is aliased with multiple different names